cyclus-files/oat/cooling-time/cooling-time-exp.xml.in

```diff
@@ -0,0 +1,1246 @@
+<simulation>
```

Comment:
Was this file generated automatically or manually?

Reply:
Manually